config/migrations/2025/20250430130307-chore--cleanup-werkingsgebieden-data/20250430153558-feat--add-werkingsgbied-for-borsbeek-district.sparql

```diff
@@ -0,0 +1,24 @@
+PREFIX geo: <http://www.opengis.net/ont/geosparql#>
+PREFIX ext: <http://mu.semte.ch/vocabularies/ext/>
+PREFIX mu: <http://mu.semte.ch/vocabularies/core/>
+PREFIX besluit: <http://data.vlaanderen.be/ns/besluit#>
+
+INSERT {
+  GRAPH <http://mu.semte.ch/graphs/public> {
+    ?borsbeekLocation a prov:Location;
+                      mu:uuid ?uuid ;
+                      rdfs:label "Borsbeek" ;
+                      ext:werkingsgebiedNiveau "District" ;
+                      geo:sfWithin ?antwerpLocation .
```

Comment:
You miss `?antwerpLocation` in your `SELECT`, causing that triple not to be inserted!